### DISCUSSION TITLE
XML External Entity (XEE) Prevention - Security vulnerabilities

### DISCUSSION THREAD

**Discussion by taskinkemal** [ZUGFeRD/mustangproject#556] [General]
Hi,

I'd like ask your opinion / perspective on XEE vulnerabilities:

https://cheatsheetseries.owasp.org/cheatsheets/XML_External_Entity_Prevention_Cheat_Sheet.html#java

Do you have an agenda to work towards this direction?

**Comment by jstaerk** [comment]
no, does anybody have a proof of concept?
